runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1026,7 +1026,7 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
             endNodeIndex, calltarget->_calleeMethod, comp());
 
       TR::Block * * blocks =
-            (TR::Block * *) comp()->trMemory()->allocateStackMemory(maxIndex
+             (TR::Block * *) comp()->trMemory()->allocateStackMemory(maxIndex
```

Comment:
can you take the space out from between the stars? it just looks odd (I know it is what was there before...